services/ui-src/serverless.yml

```diff
@@ -48,10 +43,6 @@ custom:
       items:
         - "/*"
   scripts:
-    commands:
-      testUnit: |
-        set -e
-        yarn run coverage
```

Comment:
Part of changes to close #263 
We no longer use serverless as part of running unit tests